.rubocop_todo.yml

```diff
@@ -80,10 +81,12 @@ Metrics/LineLength:
 Metrics/MethodLength:
   Max: 56
 
-# Offense count: 1
+# Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
   Max: 472
```

Comment:
ok @differentreality , then @BelieveC please change it again. And sorry for the mess :confounded: